docs/manifolds.rst

```diff
@@ -7,8 +7,14 @@ Manifolds
 All manifolds share same API. In order not to duplicate the same information, the complete public API is provided only for :class:`geoopt.manifolds.Manifold` in the end of this file.
 
 .. automodule:: geoopt.manifolds
+<<<<<<< HEAD
```

Comment:
oh god

Reply:
> oh god

Funny, given that I haven't touched any `.rst`.
I guess this could be a consequence of rebasing (I could've branched before one)